internal/services/custom_rules/service.go

```diff
@@ -60,55 +71,48 @@ func (s *Service) setCustomRules() {
 	}
 
 	for index := range customRules {
-		s.validateAndParseCustomRule(index, customRules)
+		s.validateAndParseCustomRule(&customRules[index])
```

Comment:
for index := range customRules {
		s.validateAndParseCustomRule(customRules[index])
	}

Reply:
Done.